src/main/java/dk/cngroup/lentils/service/StatusService.java

```diff
@@ -46,4 +51,17 @@ private void saveNewStatus(Status status, CypherStatus newStatus) {
         status.setCypherStatus(newStatus);
         statusRepository.save(status);
     }
+
+    public CypherStatus getCypherStatusForTeam(Cypher cypher, Team team) {
+        Status status = statusRepository.findByTeamAndCypher(team, cypher);
+        return status.getCypherStatus();
+    }
+
+    public void initializeStatusForTeamAndCypher(Cypher cypher, Team team) {
+        Status status = new Status();
+        status.setCypher(cypher);
+        status.setTeam(team);
+        status.setCypherStatus(null);
+        statusRepository.save(status);
```

Comment:
Tady si teď nejsem jistý, jestli by se neměl řádek vložit jen pokud neexistuje.

Reply:
mam to teda odstranit nebo to mergneme a ja naimplementuju ten novy koncept?